components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MaterialButton from '@material-ui/core/Button';
+import { styled } from '@material-ui/core/styles';
+
+const StyledButton = styled(MaterialButton)({
+  color: 'red',
+});
+
+
+const Button = ({ children, ...others }) => (
+  <StyledButton {...others}>
+    {children}
+  </StyledButton>
```

Comment:
We can provide additional components or wrappers to modify the default component if we want. Something like this could actually just be simplified to something like this:
`
const Button = (props) => <StyledButton {...props} />
`
The essential thing is that we spread the props from the parent to the child.